app/models/contact.rb

```diff
@@ -0,0 +1,13 @@
+class Contact < AbstractDocument
+  def self.tag_metadata_keys
+    %w(
+      contact_group
+    )
+  end
+
+  def self.metadata_name_mappings
+    {
+      "contact_group" => "Topic",
```

Comment:
[Here's an example of real usage](https://www.gov.uk/government/organisations/hm-revenue-customs/contact) - topic is the "area of business you want to contact the org about" (or thereabouts). So topic is a correct word here, if an overloaded one. We'd need an equivalent phrase.